cmd/plugins_test.go

```diff
@@ -28,24 +31,59 @@ import (
 var initChan = make(chan struct{}, 256)
 var testDirRoot string
 
+const (
+	rootDir   string = "./"
+	prefixDir string = "plugin_test_tempdir"
+)
+
 // makeDirWithSharedObjects creates a new temporary directory containing files under the runtime directory
 // it compiles all .go files into shared object files with extension ext in the corresponding directory
 // It returns the root of the directory and a cleanup function.
-func makeDirWithSharedObjects(files map[string]string, ext string) (root string, cleanup func()) {
-	root, cleanup, err := test.MakeTempFS("./", "plugin_test_tempdir", files)
+//
+// Code is duplicated from MakeTempFS() due to https://github.com/open-policy-agent/opa/issues/1185
+func makeDirWithSharedObjects(files map[string]string, ext string) (tempRootDir string, cleanup func()) {
+
+	tempRootDir, err := ioutil.TempDir(rootDir, prefixDir)
 	if err != nil {
 		panic(err)
 	}
+	cleanup = func() {
+		if err := os.RemoveAll(tempRootDir); err != nil {
+			fmt.Println("Failed to cleanup directory: ", tempRootDir)
```

Comment:
❓ Would it be interesting to include `err`? If so, we could do
```suggestion
			fmt.Println("Failed to cleanup directory %q: %v", tempRootDir, err)
```
if not, we could change the `if` to
```go
if os.RemoveAll(tempRootDir) != nil {
```

Reply:
Done but used fmt.Printf